gdx/src/com/badlogic/gdx/graphics/g3d/loader/ObjLoader.java

```diff
@@ -1,12 +1,12 @@
 /*******************************************************************************
  * Copyright 2011 See AUTHORS file.
- * 
```

Comment:
IntelliJ is removing this.

Reply:
sorry i don't use IntelliJ, there might be some configuration for this. So better ask community in Discord